tests/accessibility_tests/common_steps/accessibility.py

```diff
@@ -0,0 +1,31 @@
+
+import re
+from playwright.sync_api import Page
+from axe_core_python.sync_playwright import Axe
+import json
+import pytest
+import os
+
+@pytest.fixture()
+def test_accessibility(page: Page):
+    # Ensures run after test
+    yield
+
+    directory = "tests/accessibility_tests/accessibility_output"
+    if not os.path.exists(directory):
+        os.makedirs(directory, exist_ok=True)
+
+    axe = Axe()
+    results = axe.run(page)
+
+    wcag_violations = [violation for violation in results['violations'] if any(tag in violation['tags'] for tag in ["wcag2a", "wcag2aa"])]
```

Comment:
Can we specify which tags we want to check for separately from this list comprehension?

This would separate configuration from implementation.